include/model.php

```diff
@@ -467,10 +287,10 @@ public function get_filtered_taxonomies_query_vars() {
 	 *
 	 * @phpstan-return int<0, max>
 	 */
-	public function term_exists( $term_name, $taxonomy, $parent, $language ) {
+	public function term_exists( string $term_name, string $taxonomy, int $parent, $language ): int {
```

Comment:
Same

Reply:
Reverted in [1b6f7ed](https://github.com/polylang/polylang/pull/1503/commits/1b6f7ed2f498c60d999730b027609e01bfcffd28).